### DISCUSSION TITLE
Is WALRUS ready for use?

### DISCUSSION THREAD

**Discussion by mattWalters0** [supabase/supabase#3677] [Questions] [answered]
Is it possible to use the [WALRUS](https://github.com/supabase/walrus) utility for real-time row-level security as it is now?

There are some instructions on how to install it (execute the SQL) but it's not really clear how to use it?\

Thanks!

**Comment by megacherry** [comment]
yes, i also want to know that because it would be awesome. thanks for asking.

**Comment by silentworks** [comment, marked as answer]
This library is still being worked on, I wouldn't suggest for you to try and use it just as yet (this is probably the reason there are no instructions on how to use as yet).

**Comment by w3b6x9** [comment]
@mattWalters0 WALRUS has been incorporated into Realtime so now you can use it in any Supabase project automatically.
